service/owner/service.go

```diff
@@ -20,6 +20,7 @@ type Service struct {
 func NewService(accts accounts.Wallet, logger *log.Logger) *Service {
 	return &Service{
 		accounts: accts,
+		logger:   logger,
```

Comment:
request = {
    "rawTx": "Test",
    "address": ""
}
resp = rpc_call('owner.SignWithAddress', request) , was able to shut down the nodes as the error logger was empty